ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/contentassist/BuildPropertiesContentAssistProcessor.java

```diff
@@ -14,14 +14,18 @@
 
 import java.lang.reflect.Field;
 import java.util.ArrayList;
+import org.eclipse.jface.resource.ImageDescriptor;
 import org.eclipse.jface.text.*;
 import org.eclipse.jface.text.contentassist.ICompletionProposal;
 import org.eclipse.pde.internal.build.IBuildPropertiesConstants;
+import org.eclipse.pde.internal.ui.PDEPluginImages;
 import org.eclipse.pde.internal.ui.editor.PDESourcePage;
+import org.eclipse.swt.graphics.Image;
 
 public class BuildPropertiesContentAssistProcessor extends TypePackageCompletionProcessor {
 
 	protected PDESourcePage fSourcePage;
+
```

Comment:
extra line

Reply:
This line is introduced when you format the file and was missing before.
Adding it now is fine (so in the future others are not bother by unrelated extra lines). :)